### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by TPelikan** [lencx/ChatGPT#1080]
### Non-ChatGPT bug

- [X] This issue does not occur on chat.openai.com and only occurs on this app.

### Version

v1.1.0

### Bug description

When I try to drag file to Advanced Data Analysis the app gets stuck at "Add anything" screen and it's not possible to add anything. It works using the plus button but not by dragging from Finder.

![image](https://github.com/lencx/ChatGPT/assets/146704230/4c48890c-509d-4094-a1ae-cdb89b755041)

### OS

macOS Ventura 13.1 (22C65)

**Comment by asheroto** [comment]
Please change the title to something more descriptive. 😊

**Comment by hruzgar** [comment]
I have the same issue on my pc using the web app. Firefox and Edge both seem to have the same issue
<img width="813" alt="2023-11-02_10-31-38" src="https://github.com/lencx/ChatGPT/assets/84530266/028048c7-f3ae-44f7-b8d6-e3583526a5ce">

**Comment by hruzgar** [comment]
This happens when I want to add an picture to a normal gpt-4 chat. Then this "Add anything" screen comes there. And even opening chatgpt in different browsers doesnt change it. But after a few minutes somehow it disappears again. But when I try to add a picture to the chat the same thing happens all over again

**Comment by asheroto** [comment]
So on **Windows** with the [lencx/ChatGPT](https://github.com/lencx/ChatGPT) app, I don't see "Add anything" at all when I drag something in, using `v1.1.0`. So that itself seems like a bug with the app, but maybe just affecting Windows?

https://github.com/lencx/ChatGPT/assets/49938263/16173756-15b5-411f-b0ab-944b7a33500a

I see nothing in the devtools console (F12) either, no errors. This appears to be a bug with the app.

But with the website via browser, it works fine...

https://github.com/lencx/ChatGPT/assets/49938263/9c2014dc-1ee7-48d8-b3c8-4dac722e4ef2

If you're having an issue with ChatGPT's site in a browser, try opening a new [Chrome profile](https://support.google.com/chrome/answer/2364824?hl=en&co=GENIE.Platform=Desktop) without any extensions and see if it works. If it does work, then it's an extension issue. If it's still not working, it could be some issue with ChatGPT/OpenAI itself. If that's the case, unfortunately that issue can't be fixed here since this app is not made by OpenAI. This app is basically just a web browser that points to ChatGPT. If the issue persists on a web browser like Chrome/Firefox, issue should be reported to ChatGPT/OpenAI. If the issue occurs on both chat.openai.com as well as the [lencx/ChatGPT](https://github.com/lencx/ChatGPT) app, then it's a ChatGPT/OpenAI issue.

If the issue only occurs on the [lencx/ChatGPT](https://github.com/lencx/ChatGPT) app and not on chat.openai.com, then it's an app issue that can be resolved on this repo. 😊

**Comment by hruzgar** [comment]
Yeah I somehow thought that this repo is connected with openai. I later found out that this is just a wrapper around the web app. I'll ask in the openai community. Thank you so much for the clarification and the help!

**Comment by asheroto** [comment]
No problem, many people make that same mistake as it's a bit confusing. They added this to the top of the Issues to help, but it's easier looked over...

![image](https://github.com/lencx/ChatGPT/assets/49938263/d0cdf0e7-3060-4a5e-8a72-503799578397)

**Comment by asheroto** [comment]
@TPelikan I'm not sure if the issue you describe is related to this app or not. Can you please confirm that it is not happening for you on chat.openai.com?

If it doesn't happen there but does happen on this app, the Issue should remain open and investigated by the devs. Otherwise if it happens on your browser too then it's an OpenAI bug and the issue should be closed as it's not something the devs can fix.

**Comment by TPelikan** [comment]
@asheroto The issue is related to the app. On [chat.openai.com](https://chat.openai.com/) it works just fine for me.

**Comment by asheroto** [comment]
Got it. So on Mac your 'Add anything' screen is sticking, and on Windows, it's not showing up at all it looks like.
